### ISSUE TITLE
add a github page for demo

### ISSUE THREAD

**Issue by tigercosmos** [google/tie#552]
no need to clone repo to play it

**Comment by seanlip** [comment]
Hi @tigercosmos, thanks for the suggestion. We aren't planning to do this, since it'll be something that needs to be maintained and kept up to date. In addition, the installation steps are (we hope!) straightforward enough, and they shouldn't be too hard to follow.

That said, did you encounter any issues cloning the repo or downloading the zip file? If so, please let us know!
